edb/lang/graphql/types.py

```diff
@@ -656,7 +664,8 @@ def get_field_type(self, name, argsmap=None):
         fkey = get_fkey(name, argsmap)
         target = None
 
-        assert isinstance(name, str)
+        if not isinstance(name, str):
+            raise Exception(f"field name must be a string")
```

Comment:
f"field name must be a str, got {type(name).__name__}"

Reply:
OK.